<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="jiangguo" ignored="0" name="Test" tests="3" failures="1" timestamp="24 六月 2020 14:34:54 GMT" time="0.037" errors="0">
  <testcase name="testCase1" time="0.01" classname="com.lujiatao.jenkinstest.JenkinsTest"/>
  <testcase name="testCase2" time="0.002" classname="com.lujiatao.jenkinstest.JenkinsTest"/>
  <testcase name="testCase3" time="0.001" classname="com.lujiatao.jenkinstest.JenkinsTest">
    <failure type="java.lang.AssertionError" message="expected [99] but found [100]">
      <![CDATA[java.lang.AssertionError: expected [99] but found [100]
at com.lujiatao.jenkinstest.JenkinsTest.testCase3(JenkinsTest.java:24)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testCase3 -->
</testsuite> <!-- Test -->
